CoreVisualSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/visual/concept/VisualConceptNGTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
```

Comment:
Just noticed the year could be updated when you get a chance? I missed this on your other PR's so happy for this to be done from new PR's given we need to do search replace at some point anyway.

Reply:
Good catch. I've updated it for future PRs but yeah, given a PR is likely to be created at some stage to update it across the codebase at some stage, I'll hold off provided it's not created before this PR gets merged in